### PR TITLE
Issue #8151: Update XpathRegressionNoWhitespaceAfter to have atleast 2 tests

### PR DESCRIPTION
**Pull request by Gaurav-Punjabi**
Resolves issue #8151

### CODE REVIEW

**Comment by Gaurav-Punjabi** [comment]
@esilkensen Please review.

**Comment by strkkk** [comment]
@Gaurav-Punjabi There is a conflict with master (because of other PR). please rebase and resolve conflicts

**Comment by Gaurav-Punjabi** [comment]
> @Gaurav-Punjabi There is a conflict with master (because of other PR). please rebase and resolve conflicts

Done. 👍

File: config/checkstyle_non_main_files_suppressions.xml [hunk 1]
```diff
@@ -142,7 +142,7 @@
     <suppress  id="numberOfTestCasesInXpath"
                files="src[\\/]it[\\/]java[\\/]org[\\/]checkstyle[\\/]suppressionxpathfilter[\\/]XpathRegressionNestedTryDepthTest.java"/>
     <suppress  id="numberOfTestCasesInXpath"
-               files="src[\\/]it[\\/]java[\\/]org[\\/]checkstyle[\\/]suppressionxpathfilter[\\/]XpathRegressionNoWhitespaceAfterTest.java"/>
+               files="src[\\/]it[\\/]java[\\/]org[\\/]checkstyle[\\/]suppressionxpathfilter[\\/]XpathRegressionNoWhitespaceBeforeTest.java"/>
```
Comment (strkkk) [inline config/checkstyle_non_main_files_suppressions.xml:145]: `XpathRegressionNoWhitespaceBeforeTest` it was merged, there is something wrong with your conflict resolution

Comment (Gaurav-Punjabi) [reply config/checkstyle_non_main_files_suppressions.xml]: Oh yes, fixed it.
Please check

**Pull request merged**